console/console-init/ui/src/graphql-module/queries/project.ts

```diff
@@ -4,15 +4,62 @@
  */
 
 import gql from "graphql-tag";
+import { IProjectFilter } from "modules/project/ProjectPage";
+import { generateFilterPattern } from "./query";
+import { removeForbiddenChars } from "utils";
 
-const FILTER_RETURN_PROJECTS = (projectFilterParams?: any) => {
-  const { projectname } = projectFilterParams;
+const FILTER_RETURN_PROJECTS = (projectFilterParams: IProjectFilter) => {
```

Comment:
I think we don't need return in between function name. It should  be FILTER_PROJECTS